test/Sentry.Hangfire.Tests/Sentry.Hangfire.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net7.0;net6.0;net48</TargetFrameworks>
```

Comment:
do these match what we're targeting in the lib? like net6 is there to cover ns2.1, but ns2.0 has no coverage?